templates/docs/rhpam710-authoring-ha.adoc

```diff
@@ -516,57 +516,19 @@ kieserver-app-secret
 
 Clustering in OpenShift EAP is achieved through one of two discovery mechanisms:
 Kubernetes or DNS. This is done by configuring the JGroups protocol stack in
-standalone-openshift.xml with either the `<openshift.KUBE_PING/>` or `<openshift.DNS_PING/>`
```

Comment:
this needs to be c hanged on the jboss-kie-modules repo on the temp[alte adoc.
templates' adoc files are generated, no need to update it manually.

Reply:
Ok